_data/sites.json

```diff
@@ -6414,6 +6444,16 @@
         ]
     },
 
+    {
+        "name": "Uncubed",
+        "url": "https://uncubed.com/learn/users/edit",
+        "difficulty": "hard",
+        "notes": "You can deactivate your account from 'Settings', but must contact them to terminate your account",
```

Comment:
What contact method do users need to use?

Reply:
I have updated the notes to explain that the user must contact the company via email and have added a corresponding email.